.pytool/Plugin/DscCompleteCheck/DscCompleteCheck.py

```diff
@@ -95,11 +96,18 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
         dp.SetInputVars(environment.GetAllBuildKeyValues())
         dp.ParseFile(wsr_dsc_path)
 
+        def module_in_dsc(inf: str) -> bool:

```

Comment:
This function is declared in the middle of RunBuildPlugin

Could it be moved into its own function as part of the DscCompleteCheck() class?

Reply:
This can be a bit subjective but based on the length of the `RunBuildPlugin()` function, I could see a case made for it being more readable as a private class function.

Reply:
This is all subjective and I have no strong opinions on the matter. If you'd like it moved out to a private function of the `DscCompleteCheck` class, I have no objections. I can see arguments for readability either way.

I declared the function inside `RunBuildPlugin` because it a) made the arguments cleaner by capturing `dp` and b) was only needed for the `RunBuildPlugin` function.

Reply:
Moved.